src/Controls/src/Core/Shell/ShellItem.cs

```diff
@@ -109,10 +109,14 @@ bool IShellItemController.ShowTabs
 				// Windows supports nested tabs so we want the tabs to display
 				// if the current shell section has multiple contents
 				if (ShellItemController.GetItems().Count > 1 ||
-					(CurrentItem as IShellSectionController).GetItems().Count > 1)
+					(CurrentItem != null && (CurrentItem as IShellSectionController).GetItems().Count > 1))
```

Comment:
I think we can just add some null terminator to make this work

`(CurrentItem as IShellSectionController)?.GetItems()?.Count > 1)`